tests/UI/campaigns/functional/FO/02_headerAndFooter/01_checkLinksInHeader.js

```diff
@@ -0,0 +1,184 @@
+  });
+
+  it('should check sign out link', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkSignOutLink', baseContext);
+
+    // Sign out
+    await myAccountPage.logout(page);
+
+    const pageTitle = await loginPage.getPageTitle(page);
+    await expect(pageTitle).to.equal(loginPage.pageTitle);
+  });
+
+  it('should check shopping cart link', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkShoppingCartLink', baseContext);
+
+    await loginPage.goToHomePage(page);
+
+    // Add product to cart by quick view
+    await homePage.addProductToCartByQuickView(page, 1, 1);
+
+    // Close block cart modal
+    const isQuickViewModalClosed = await homePage.closeBlockCartModal(page);
+    await expect(isQuickViewModalClosed).to.be.true;
+
+    // Check cart link
+    await homePage.goToHeaderLink(page, 'Cart');
+
+    const pageTitle = await cartPage.getPageTitle(page);
+    await expect(pageTitle).to.equal(cartPage.pageTitle);
+  });
+
+  it('should check logo link', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkLogoLink', baseContext);
+
+    await homePage.goToHeaderLink(page, 'Logo');
+
+    const pageTitle = await homePage.getPageTitle(page);
+    await expect(pageTitle).to.equal(homePage.pageTitle);
+  });
+
+  [Categories.clothes, Categories.accessories, Categories.art].forEach((test) => {
+    it(`should check category '${test.name}' link`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `check${test.name}Link`, baseContext);
+
+      await homePage.goToCategory(page, test.id);
+
+      const pageTitle = await homePage.getPageTitle(page);
+      await expect(pageTitle).to.equal(test.name);
+    });
+  });
+
+  [
+    {args: {category: Categories.clothes, subcategory: Categories.men}},
+    {args: {category: Categories.clothes, subcategory: Categories.women}},
+    {args: {category: Categories.accessories, subcategory: Categories.stationery}},
+    {args: {category: Categories.accessories, subcategory: Categories.homeAccessories}},
+  ].forEach((test) => {
+    it(`should check subcategory '${test.args.subcategory.name}' link`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `check${test.args.subcategory.name}Link`, baseContext);
+
+      await homePage.goToSubCategory(page, test.args.category.id, test.args.subcategory.id);
+
+      const pageTitle = await homePage.getPageTitle(page);
+      await expect(pageTitle).to.equal(test.args.subcategory.name);
+    });
+  });
+
+  it('should check search link', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkSearchLink', baseContext);
+
+    await homePage.searchProduct(page, Products.demo_8.name);
+
+    const pageTitle = await searchResultsPage.getPageTitle(page);
+    await expect(pageTitle).to.equal(searchResultsPage.pageTitle);
+  });
```

Comment:
Maybe this part should be on a different test, wdyt ?

Reply:
I just automated the steps of exploratory tests on test rail :
Check all the links in the header :
contact us
sign in
logo
all the menu and sub-menu links

Reply:
could be seperated on testrail too